RELEASE-NOTES.txt

```diff
@@ -1,5 +1,9 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 
+20.8
+----
+* [**] Added author button to the post settings and the author of the post can be changed with it.
```

Comment:
Are you going to add PR url here?

Reply:
Oh yes, I forgot that.
Added: 2e842c2509cc92b977fbe48ba8ce029b1e7980b3